.env.example

```diff
@@ -2,7 +2,7 @@ NODE_ENV=test
 KEYBASE_ENV=devnet # testnet | devnet | mainnet
 NODE_CONFIG_DIR="src/config"
 
-# ELROND configs
```

Comment:
elrond api : https://testnet-api.multiversx.com/
gateway: https://testnet-gateway.multiversx.com
private: https://testnet-private-api.elrond.com (here I'm not sure but you can check or remove the value

Reply:
Updated